source/portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -506,6 +510,8 @@ NetworkInterface_t * pxSAM_FillInterfaceDescriptor( BaseType_t xEMACIndex,
     pxInterface->pfOutput = prvSAM_NetworkInterfaceOutput;
```

Comment:
When setting the pcName , it can be set as 
```
snprintf( pcName, sizeof( pcName ), "GMAC%u", ( unsigned ) xEMACIndex );
pxInterface->pcName = pcName;
```
This method is being followed for STM32Hxx port, and might be more useful for debugging purpose.

Reply:
Updated